scripts/download.sh

```diff
@@ -34,6 +34,22 @@ else
   echo "Snapshot already present"
 fi
 
-if [ ! -z "$STATS_IP" ]; then
-  echo -n "ctlstore.reflector.init_snapshot_download_time:$(($END - $START))|h|#$TAGS" | nc -u -w1 $NODE_IP $STATS_PORT
-fi
\ No newline at end of file
+emit_metrics(){
+  if [ ! -z "$STATS_IP" ]; then
+    counter=0
+    while ! echo exit | nc -u -w1 $NODE_IP $STATS_PORT;
```

Comment:
this `-w1` flag is actually already a 1 sec timeout. So I don't think we need it since we end up sleeping anyways

Reply:
good call